mpqp/core/instruction/measurement/pauli_string.py

```diff
@@ -762,4 +998,6 @@ def __hash__(self):
 Matrix representation:
 `\begin{pmatrix}1&0\\0&-1\end{pmatrix}`
 """
+
+PAULI_ATOM_DICT = {"I": I, "X": X, "Y": Y, "Z": Z}
```

Comment:
this seems to be for internal use only, I would put it as a private attribute of the class or something like that

Reply:
done